README.md

```diff
@@ -9,632 +11,57 @@
-* Login via OAuth is improved
-* External video source is room is fixed
-* Multiple improvements of White-board
-* Multiple improvements of Chat
-* JS/CSS files are minified and merged to reduce load time
-* Overall stability is improved
-
-Other fixes and improvements, 43 issues were fixed
-
-4.0.0
------
-[Release 4.0.0](https://archive.apache.org/dist/openmeetings/4.0.0), provides following improvements:
-
-* Room White board is rewritten to use HTML5 instead of Flash
-* All Audio/Video components were rewritten using Apache Flex
-* Bunch of automatic tests were written, code was cleaned-up and simplified
-* outdated SMSlib was removed from the dependencies
-* RTL in room is improved
-* swftool 3rd party tool is not required any more
-* OAuth authorization is fixed additional VK provider is added
-* language resources are minimized
-* performance is improved
-
-Admin->Config
-* config keys are generalized
-* config types are introduced
-
-Other fixes and improvements, 515 issues were fixed
-
-3.3.2
------
-[Release 3.3.2](https://archive.apache.org/dist/openmeetings/3.3.2), provides following improvements:
-
-* Audio/Video in conference room is fixed
-* Strong password is enforced during self registration
-* 'dashboard.show.chat' should work as expected
-* New Setting is added 'user can create rooms'
-
-Other fixes and improvements, 8 issues were fixed
-
-
-3.3.1
------
-[Release 3.3.1](https://archive.apache.org/dist/openmeetings/3.3.1), provides following improvements:
-
-* Clustering works as expected
-* SIP works as expected
-* Multiple issues in web services were fixed
-* RTMPT mode is fixed
-* Multiple UI improvements
-
-Other fixes and improvements, 19 issues were fixed
-
-
-3.3.0
------
-[Release 3.3.0](https://archive.apache.org/dist/openmeetings/3.3.0), provides following improvements:
-
-Security fixes in:
-* Chat
-* All requests via security headers
-* More secure password processing rules and storage
-* More strict rules for uploaded files
-* SQL injection in web services
-
-11 security vulnerabilities were addressed
-
-Whiteboard:
-* Room is displayed without overlap in IE
-* Multiple display issues
-* Wb room element can now be hidden
-
-Other fixes and improvements, 21 issues were fixed
-
-
-3.2.1
------
-[Service release 1 for 3.2.0](https://archive.apache.org/dist/openmeetings/3.2.1), provides following improvements:
-
-Room
-* Video is more stable
-* Office files download is fixed
-* Multi-upload is added
-* External video works as expected
-* WB drawing on slides works as expected
-
-Chat
-* chat is made resizable
-* multiple issues in chat are fixed
-* typing indicator is added
-
-Calendar
-* date/time validator is improved
-* whole group can be invited by admin to event
-
-Other fixes and improvements, 49 issues were fixed
+### Release Notes
+See the [CHANGELOG.md](/CHANGELOG.md) file for a detailed log.
 
-License
-=======
-Licensed under Apache License 2.0 - http://www.apache.org/licenses/LICENSE-2.0
+Recent Releases:
```

Comment:
I would not drop the entire content of the file, but better use `collapsible sections` (https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections)